Content.Client/Chat/Managers/ChatManager.cs

```diff
@@ -81,4 +88,19 @@ public void SendMessage(string text, ChatSelectChannel channel)
                 throw new ArgumentOutOfRangeException(nameof(channel), channel, null);
         }
     }
+
+    public void SendMessage(string text, string channel)
+    {
+        if (_prototypeManager.TryIndex(channel, out CommunicationChannelPrototype? channelPrototype))
+        {
+            SendMessage(text, channelPrototype);
+        }
+    }
+
+    public void SendMessage(string text, CommunicationChannelPrototype channel)
+    {
+        var str = text.ToString();
+
+        _consoleHost.ExecuteCommand($"chat {channel.ID} \"{CommandParsing.Escape(str)}\"");
```

Comment:
i would like to remove this in favour of direct message/event sending, launching some console commands looks ugly Q_Q
It also can cause unexpected results... but thats of none concern as we were living with this for so long...

Reply:
There is a PR by ElectroSr that does this; it would have some minor merge conflicts, but I agree it would be nice to have.

Reply:
sure, valid. i agree.